vercel/turbo
# fix: Load vercel token into command base

### Description
There's an issue where if you're logged into Vercel, and don't have a Turbo token, then multiple commands will fail (unless you run `turbo login -f`).

This happened because the command base wasn't looking for the Vercel token. This addresses that by loading in the Vercel token for use if it's present.

### Testing Instructions
1. Ensure no tokens exist for Turbo or Vercel (`vc logout && turbo logout`)
2. `vc login`
3. `turbo login` -> Existing token found!
4. `turbo link` in a repo the token has access to -> Normal link process happens
